1.9/usage/managing-services/package-api.md

```diff
@@ -0,0 +1,38 @@
+---
+post_title: DC/OS Package Management API
+nav_title: Package API
+menu_order: 10
+---
+
+The Package Management API allows you to install DC/OS services from DC/OS packages stored in a DC/OS package registry, such as the [Mesosphere Universe](/docs/1.9/overview/concepts/#mesosphere-universe).
```

Comment:
global: "The package management API" ("package management" describes the API, i.e., isn't a product name/proper noun)